tests/unit/test_environment.py

```diff
@@ -8,109 +7,106 @@
 from rocketpy import Environment
 
 
-def test_env_set_date(example_env):
-    """Test that the date is set correctly in the environment object. This
-    basically takes a date and time and converts it to a datetime object, then
-    set the date to the example environment object. The test checks if the
-    datetime object is the same as the one in the example environment object.
+def test_date_naive_set_date_saves_utc_timezone_by_default(
+    example_plain_env, example_date_naive
+):
+    """Tests time zone is set to UTC by default in environment obj given a date_naive input
 
```

Comment:
This line is too long (more than 80 columns). I know it is annoying, but can you break it in 2 please?